src/components/connectors/card/index.tsx

```diff
@@ -53,22 +63,56 @@ function ConnectorCard({ cta, docsUrl, logo, title, details }: Props) {
                     }}
                 >
                     <Box>
-                        <Box sx={connectorImageBackgroundSx}>
-                            <Stack
-                                sx={{
-                                    justifyContent: 'center',
-                                    alignItems: 'center',
-                                }}
+                        <Stack
+                            sx={{
+                                marginBottom: recommended ? 1 : 2,
+                            }}
+                        >
+                            <Box
+                                sx={
+                                    recommended
+                                        ? {
+                                              ...connectorImageBackgroundSx,
+                                              mb: 0,
+                                              borderBottomLeftRadius: 0,
+                                              borderBottomRightRadius: 0,
+                                          }
+                                        : connectorImageBackgroundSx
```

Comment:
If it is recommended we don't want the border radius on the bottom so the label can have the rounded corner.